app/helpers/simulator_helper.rb

```diff
@@ -1,2 +1,25 @@
 module SimulatorHelper
+  def return_image_file(data_url)
```

Comment:
Look at how `app/helpers/utils.rb` has been implemented. Is it possible to implement it in a similar fashion for the sake of uniformity?

Reply:
Obviously it can be placed there but i think keeping it utils sacrifices the readability.

Reply:
I agree, I'm asking to implement `SimulatorHelper` is a similar fashion as `Utils` with the helper functions as static methods inside the `SimulatorHelper` and removing the `include` from the `SimulatorController`.

Reply:
Helpers are supposed to be implemented as modules. But one downside is helpers have global scope. Since this logic is specific to simulator controller it is better to keep it in simulator controller itself as private methods

Reply:
cool